app/src/androidTest/java/org/dhis2/usescases/searchte/SearchTETest.kt

```diff
@@ -124,6 +131,27 @@ class SearchTETest : BaseTest() {
         }
     }
 
+    @Test
+    fun shouldSuccessfullyShowMapAndTeiCard() {
+        val firstName = "Gertrude"
+
+        prepareChildProgrammeIntentAndLaunchActivity()
+
+        searchTeiRobot {
+            clickOnOptionMenu()
+            clickOnShowMap()
+            waitToDebounce(1000)
```

Comment:
why do we need a sleep if we are adding idling?

Reply:
Was testing something a forgot to remove it. 🤦‍♂️

Reply:
done